src/js/view/view.js

```diff
@@ -886,7 +886,8 @@ function View(_api, _model) {
             floatingPlayer = null;
 
             if (forever) {
-                _floatOnScroll = false;
+                _floatOnScroll = null;
```

Comment:
Why `null` rather then `false`?

Reply:
It's now an object so I put `null` instead of changing it to a boolean